features/admin/company_details_edit.feature

```diff
@@ -80,8 +80,8 @@ Scenario: Admin CCS Data Controller can edit a supplier's details
   When I click the summary table 'Change' link for 'Company registration number'
   And I enter '87654321' in the 'Companies House number' field and click its associated 'Save' button
   When I click the summary table 'Change' link for 'Address'
-  And I enter '10 Downing Street' in the 'Building and street' field
-  And I enter 'London' in the 'Town or city' field
-  And I enter 'AB1 2CD' in the 'Postcode' field
-  And I enter 'United Kingdom' in the 'location-autocomplete' field and click the selected autocomplete option
+  And I enter '14 Duke Street' in the 'Building and street' field
+  And I enter 'Dublin' in the 'Town or city' field
+  And I enter 'H3 LY5' in the 'Postcode' field
+  And I enter 'Ireland' in the 'location-autocomplete' field and click the selected autocomplete option
```

Comment:
Was changing the address necessary to make the fix work?

Reply:
I didn't change the address - I put a _new_ address where there wasn't one before. I used a _different_ address so it was possible to differentiate in tests where this address was sourced from.